processor/manager.go

```diff
@@ -109,3 +112,70 @@ func New(ctx context.Context, clearDb *bool, gwDb, rtDb, brtDb, errDb *jobsdb.Ha
 	}
 	return proc
 }
+
+func (proc *Handle) cleanUpRetiredJobs() {
```

Comment:
What's the reason behind having these functions in `manager.go` instead of `processor.go`?